arcadia_pycolor/mpl.py

```diff
@@ -416,10 +444,22 @@ def load_fonts(font_folder: Union[str, None] = None) -> None:
         font_folder (str, optional): the folder to search for fonts in.
             Uses the default system font folder if None.
     """
-    for fontpath in font_manager.findSystemFonts(fontpaths=font_folder, fontext="ttf"):
-        if FONT_FILTER.lower() in fontpath.lower():
-            font_manager.fontManager.addfont(fontpath)
-            font_manager.FontProperties(fname=fontpath)
+    arcadia_font_paths = []
+
+    # On macOS, look in the standard font locations first.
+    # This is faster than matplotlib's font search implementation.
+    # See https://github.com/Arcadia-Science/arcadia-pycolor/pull/58.
+    if font_folder is None and sys.platform == "darwin":
+        arcadia_font_paths.extend(_find_macos_arcadia_fonts())
+    # If no fonts are found, fallback to full system search.
+    else:
```

Comment:
Something seems wrong here - either this comment is wrong or the else-block is wrong, because as written, we get to the else-block if we're not on a Mac, not if no fonts are found. It feels to me like the comment should be correct and instead of the else block there should be a new if-block:

```suggestion
    # If no fonts are found, fallback to full system search.
    if not arcadia_font_paths:
```

Reply:
Yup, I changed the code but forgot to update the comments. Was just about to fix this when your review came in haha.